src/UI/Component/Button/Factory.php

```diff
@@ -260,4 +260,38 @@ public function tag($label, $action);
 	 */
 	public function bulky($icon_or_glyph, $label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Toggle Button triggers the activation/deactivation of some content already shown on the screen, i.e.
+	 *       a filter. The deactivation of content means, that it is still shown and the user can still interact
+	 *       with it, but it has no effect to the system.
+	 *   composition: >
+	 *       The Toggle Button uses the primary color as background when it is activated.
+	 *       The Toggle Button uses the primary color with a grayscale filter as background when it is deactivated.
```

Comment:
This is very color focused and I would remove the naming of the color. maybe just state, that the deactivated color differs from the activated one. If we are too specific about the design, we need to talk every little less change in the JF. Also note, that the state should not only rely on color. What about the toggle, that moves from left to right? Which is which?

Reply:
Done.